Rise-Vision/rise-vision-apps
# Feature/file existence service

## Description
This extracts the file existence service to an independent service.

## Motivation and Context
- Having the service as independent, will let it be shared with rise-video; and also will help to easily implement the issue 1066 fix.
- A service for file metadata related utilities was also created ( fileMetadataUtilsService ) so this functionality can also be shared. In following PRs more functionality will be added here.
- Unit tests for both services were created. Some of these tests cover existing functionality that was not being tested before.

## How Has This Been Tested?
Image settings are still working properly here:
https://apps-stage-9.risevision.com/templates/edit/46841dc1-efef-4a2d-84c9-1e0a14932ff4/?cid=cf85e5c4-9439-40ce-94d6-e5ea6ea2411c

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
    - Manual tests ok
    - Automated tests created ( previous unit and E2E tests should still work as this is a refactoring )
    - will release next Monday
    - monitoring: this affects no data, will check editor on production after merging, and rollback if there are problems
    - Rollback is simple ( no data )
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No need to notify support
